Winget-AutoUpdate/locale/en.xml

```diff
@@ -14,7 +14,7 @@
 		</output>
 		<output id="2">
 			<!--{App} will be updated-->
-			<title>{0} will be updated!</title>
+			<title>{0} is being updated!</title>
```

Comment:
English is not my first language. Should I change at least this part of your PR ? 😅